sandbox/functions-recipes/includes/durable-activity-functions-errors.md

```diff
@@ -0,0 +1,35 @@
+## Handling activity function exceptions
+Unhandled exceptions thrown in activity functions will be rethrown in the calling orchestrator function as a `FunctionFailedException`. The `InnerException` property of the `FunctionFailedException` will contain the original exception from the activity.
+
+```csharp
+[FunctionName("PlaceOrder")]
+public static async Task OrderOrchestration(
+    [OrchestrationTrigger] DurableOrchestrationContext context,
+    TraceWriter log)
+{
+    try
+    {
+        bool reserved = await context.CallActivityAsync<bool>("CheckAndReserveInventory", null);
```

Comment:
Not a big deal, but it's a bit strange to save the result into a variable and never use that variable. Could the return value part be removed?